test/HubPool.RefundInitilization.ts

```diff
@@ -0,0 +1,66 @@
+import { expect } from "chai";
```

Comment:
nothing changed here, just moved and cut down on redundant imports/method calls.